### DISCUSSION TITLE
Small UI bug. Create plan button is not active after plan creation

### DISCUSSION THREAD

**Discussion by FlyVeryHigh** [subquery/subql#956] [Frontier TestNet - Bug Reporting]
Chrome.
1. Create a plan.
2. Try to create the new plan (without window refresh). Create plan button is inactive:
![photo_2022-05-02_15-01-37](https://user-images.githubusercontent.com/55284763/166224142-d745fb50-dceb-4348-8300-36c09c66e1a7.jpg)
3. Refresh the window. Then can create new plan.

**Comment by lilian79511** [comment]
It takes around 15s for the transaction to go through therefore there's a sand clock icon on the right. The button is disabled during this period of time.

Will see how to give better hint on this action though, thanks for reporting.

**Reply by FlyVeryHigh** [reply]
Recentrly tried again and now it is ok. In the past the button was inactive even 1 minute afer tx confirm.
